frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -505,112 +536,128 @@ const PersonForm = (props: Props) => {
             />
           )}
         </div>
-        <div className="usa-form">
-          <Select
-            label={t("patient.form.contact.country")}
-            name="country"
-            value={patient.country || "USA"}
-            options={countryOptions}
-            onChange={onPersonChange("country")}
-            onBlur={() => {
-              onBlurField("country");
-            }}
-            validationStatus={validationStatus("country")}
-            errorMessage={errors.country}
-            required
-          />
+        <div className="sr-patient-sub-header">
+          {t("patient.form.contact.addressHeading")}
         </div>
-        <div className="usa-form">
-          <Input
-            {...commonInputProps}
-            field="street"
-            label={t("patient.form.contact.street1")}
-            required
-          />
-        </div>
-        <div className="usa-form">
-          <Input
-            {...commonInputProps}
-            field="streetTwo"
-            label={t("patient.form.contact.street2")}
-          />
-        </div>
-        <div className="usa-form">
-          <Input
-            {...commonInputProps}
-            field="city"
-            label={t("patient.form.contact.city")}
-            required
-          />
-          {view !== PersonFormView.SELF_REGISTRATION && (
-            <Input
-              {...commonInputProps}
-              field="county"
-              label={t("patient.form.contact.county")}
-            />
-          )}
-          {patient.country === "USA" ? (
-            <div className="grid-row grid-gap">
-              <div className="mobile-lg:grid-col-6">
-                <Select
-                  label={t("patient.form.contact.state")}
-                  name="state"
-                  value={patient.state || ""}
-                  options={stateCodes.map((c) => ({ label: c, value: c }))}
-                  defaultOption={t("common.defaultDropdownOption")}
-                  defaultSelect
-                  onChange={onPersonChange("state")}
-                  onBlur={() => {
-                    onBlurField("state");
-                  }}
-                  validationStatus={validationStatus("state")}
-                  errorMessage={errors.state}
-                  required
-                />
-              </div>
-              <div className="mobile-lg:grid-col-6">
-                <Input
-                  {...commonInputProps}
-                  field="zipCode"
-                  label={t("patient.form.contact.zip")}
-                  required
-                />
-              </div>
+        <Checkbox
+          id={"unknownAddress"}
+          name={"unknownAddress"}
+          label={t("patient.form.contact.unknownAddress")}
+          checked={patient.unknownAddress ?? undefined}
+          onChange={(e) => {
+            onPersonChange("unknownAddress")(e.target.checked);
+          }}
+        />
+        {!patient.unknownAddress && (
```

Comment:
Can this block of code be moved to its own component?. Similar how the phone number is its own component.